### PR TITLE
Feature/form error handling

### LINKED ISSUE #70: Form Error Handling
`NewContact.js`
-all fields complete
-valid phone number

`Form.js`
-all fields complete
-valid address

### PR DESCRIPTION
**Pull request by carolineshowscode**
## Type of Change
- [x] JSX
- [ ] HTML
- [ ] JavaScript
- [x] CSS
- [x] React
- [ ] React Router
- [ ] Cypress
- [ ] Apollo
- [ ] GraphQL
- [ ] Twilio API
- [ ] Refactor/Fix Bugs

## Please give a short description of the changes that were made below

- Fix:

- Feature:
-Main Page `Form` error handling: all fields must be complete, valid address entered
-`AddContact` form error handling: add fields must be complete, valid phone number entered
-`AddContact` CSS update

-To Do: Add Contact Alert

## How has this been tested?
- [x] In Browser
- [x] In Console
- [ ] Cypress
- [ ] Postman

### CODE REVIEW

File: src/Components/AddContact/AddContact.js [hunk 1]
```diff
@@ -26,8 +28,41 @@ function AddContact({ user }) {
 
   function addContact(e) {
     e.preventDefault();
+    if(!firstName || !lastName || !countryCode || !areaCode || !phoneNumber){
```
Comment (pcmueller) [inline src/Components/AddContact/AddContact.js:31]: Now that's a conditional :)

File: src/Components/AddContact/AddContact.js [hunk 2]
```diff
@@ -26,8 +28,41 @@ function AddContact({ user }) {
...
+      return
+    }
+    setValidCheck(false);
+    setPhoneVerify(false);
     let number = `+${countryCode}${areaCode}${phoneNumber}`;
     createContact( {variables: { firstName: firstName, lastName: lastName, phoneNumber: number}});
+    clearForm();
+  }
+
+  function checkPhoneNum() {
```
Comment (pcmueller) [inline src/Components/AddContact/AddContact.js:44]: Nice work verifying the phone number length!

File: src/Components/Form/Form.js [hunk 3]
```diff
@@ -69,8 +86,15 @@ function Form({ contacts, handleEtaChange, userInfo }) {
     e.preventDefault();
   }
 
+
+
   return (
-    <form className='trip-form' onSubmit={handleSubmit}>
+    <form className='trip-form' onSubmit={handleSubmit} >
+    {valid && <p>Complete form fields with Valid Data</p>}
```
Comment (pcmueller) [inline src/Components/Form/Form.js:93]: Error handling looks solid.  Where exactly does the message render?  Does this appear within the form?

**Review by pcmueller — APPROVED** [review]
Great job with all the error handling!

**Review by foymikek — APPROVED** [review]
Nice error handling, looks clean!

**Pull request merged**